sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/CreateNotification.groovy

```diff
@@ -70,14 +71,16 @@ class CreateNotification implements CreateNotificationInput {
         int failedQCCount = filterSamplesByStatus(project.sampleCodes, "SAMPLE_QC_FAIL").size()
         int availableDataCount = filterSamplesByStatus(project.sampleCodes, "DATA_AVAILABLE").size()
 
+        if (noRelevantStatusWasUpdated(failedQCCount, availableDataCount)) {
```

Comment:
```suggestion
        if (!isRelevantStatusUpdate(failedQCCount, availableDataCount)) {
```

Reply:
NIT: I think a shorter title makes the method easier to read. With adding words like 'was' you add fillwords that make the method name even longer and harder to graps. 
I think we can neglegt the "no" by using '!' which also shortens the wording. But I leave it to you how to implement it in the end

Reply:
Good point thanks 👍